lnrpc/lightning.proto

```diff
@@ -2217,6 +2217,11 @@ message OpenChannelRequest {
     to use the default fee rate value specified in the config or 0.
     */
     bool use_fee_rate = 24;
+
+    /*
+    The number of satoshis we require the remote peer to reserve.
```

Comment:
Should we mention here that the reserve needs to be above the dust limit and below 20%?